File: programming files/rgb_to_hex.py
def rgb_to_hex(r, g, b):
    # create the hex value for red
    r = max(0, min(255, r))
    # create the hex value for green
    g = max(0, min(255, g))
    # create the hex value for blue
    b = max(0, min(255, b))
    # combine all hex values in a string and return the value
    return '{:02X}{:02X}{:02X}'.format(r, g, b)


# test with hex_color = rgb_to_hex(255, 127, 0) # returns "FF7F00"

File: programming files/rgb_to_hex.java
public class rgb_to_hex {
    public static void main(String[] args) {
        //Set values
        int r = 255;
        int g = 127;
        int b = 0;

        //Calls function
        String hexColor = rgbToHex(r, g, b);
        //Prints out the values
        System.out.println("RGB color (" + r + ", " + g + ", " + b + ") = " + hexColor);
    }
    //Functiont that converts the rgb values to the hex values
     public static String rgbToHex(int r, int g, int b) {
        r = Math.min(255, Math.max(0, r));
        g = Math.min(255, Math.max(0, g));
        b = Math.min(255, Math.max(0, b));

         //returns the values
        return String.format("%02X%02X%02X", r, g, b);
    }
}

//Test with RGB color (255, 127, 0) = FF7F00


